types/api/setup-intents.d.ts

```diff
@@ -131,17 +131,30 @@ export namespace SetupIntent {
   }
 
   export interface NextAction {
-    redirect_to_url?: NextAction.RedirectToUrl;
-
     /**
-     * Type of the next action to perform, one of `redirect_to_url` or `use_stripe_sdk`.
+     * Type of the next action to perform, one of `redirect_to_url`, `use_stripe_sdk`, `alipay_handle_redirect`, `oxxo_display_details`, or `verify_with_microdeposits`.
```

Comment:
my assumption here is that these additional types (like `alipay_handle_redirect` and `oxxo_display_details`) do not have their own option hashes?